src/test/java/collections/phonebook/PhoneBookTest.java

```diff
@@ -0,0 +1,49 @@
+package collections.phonebook;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PhoneBookTest {
+    private static Map<String, List<String>> records;
```

Comment:
це ти тестуєш Map, а не свою реалізацію PhoneBook. 

```suggestion
    private PhoneBook phoneBook = new PhoneBook();
```